<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountForm</class>
 <widget class="QWidget" name="AccountForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="MarvieController.qrc">:/MarvieController/icons/icons8-user-96-blue.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="accountLabel">
           <property name="minimumSize">
            <size>
             <width>47</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="accountNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="minimumSize">
            <size>
             <width>47</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="accountPasswordEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="rememberMeCheckBox">
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logInButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MarvieController.qrc"/>
 </resources>
 <connections/>
</ui>
